app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,31 @@
+class UserPresenter < SimpleDelegator
+  include ActionView::Helpers::FormOptionsHelper
+
+  delegate :global, to: :user_roles, prefix: true
+
+  def self.wrap(users)
+    users.map { |user| new(user) }
```

Comment:
I'm not sure it's going to be an issue here, but it's sometimes nice to have these kind of things be lazy. Since `users` is likely an AR::Relation, it's lazy, it would be nice to keep this wrapper function lazy as well so it doesn't hit the database until it absolutely has to.